src/main/java/io/github/thebusybiscuit/exoticgarden/items/FoodRegistry.java

```diff
@@ -1,531 +1,617 @@
+        new ItemStack[] {getItem("CHOCOLATE_BAR"), new ItemStack(Material.SUGAR), SlimefunItems.WHEAT_FLOUR, SlimefunItems.BUTTER, SlimefunItems.HEAVY_CREAM, new ItemStack(Material.EGG), null, null, null},
+        13)
+        .register(plugin);
+
+        new CustomFood(food, new SlimefunItemStack("BOSTON_CREAM_PIE", "dfd71e20fc50abf0de2ef7decfc01ce27ad51955759e072ceaab96355f594f0", "&rBoston Cream Pie", "", "&7&oRestores &b&o" + "4.5" + " &7&oHunger"),
+        new ItemStack[] {null, getItem("CHOCOLATE_BAR"), null, null, SlimefunItems.HEAVY_CREAM, null, null, getItem("BISCUIT"), null},
+        9)
+        .register(plugin);
+
+        new CustomFood(food, new SlimefunItemStack("HOT_DOG", "15a995f41c4630e879f34dbac41f92317558534f2d4d7a468b670e1dc52177", "&rHot Dog", "", "&7&oRestores &b&o" + "5.0" + " &7&oHunger"),
+        new ItemStack[] {null, null, null, null, new ItemStack(Material.COOKED_PORKCHOP), null, null, new ItemStack(Material.BREAD), null},
+        10)
+        .register(plugin);
+
+        new CustomFood(food, new SlimefunItemStack("BACON_WRAPPED_CHEESE_FILLED_HOT_DOG", "15a995f41c4630e879f34dbac41f92317558534f2d4d7a468b670e1dc52177", "&rBacon wrapped Cheese filled Hot Dog", "&7&o\"When I chef\" - @Eyamaz", "", "&7&oRestores &b&o" + "8.5" + " &7&oHunger"),
+        new ItemStack[] {getItem("BACON"), getItem("HOT_DOG"), getItem("BACON"), null, getItem("CHEESE"), null, null, null, null},
+        17)
+        .register(plugin);
+
+        new CustomFood(food, new SlimefunItemStack("BBQ_BACON_WRAPPED_HOT_DOG", "15a995f41c4630e879f34dbac41f92317558534f2d4d7a468b670e1dc52177", "&rBBQ Bacon wrapped Hot Dog", "&7&o\"wanna talk about hot dogs?\" - @Pahimar", "", "&7&oRestores &b&o" + "8.5" + " &7&oHunger"),
+        new ItemStack[] {getItem("BACON"), getItem("HOT_DOG"), getItem("BACON"), null, getItem("BBQ_SAUCE"), null, null, null, null},
+        17)
+        .register(plugin);
+
+        new CustomFood(food, new SlimefunItemStack("BBQ_DOUBLE_BACON_WRAPPED_HOT_DOG_IN_A_TORTILLA_WITH_CHEESE", "15a995f41c4630e879f34dbac41f92317558534f2d4d7a468b670e1dc52177", "&rBBQ Double Bacon wrapped Hot Dog in a Tortilla with Cheese", "&7&o\"When I chef\" - @Eyamaz", "", "&7&oRestores &b&o" + "10.0" + " &7&oHunger"),
+        new ItemStack[] {getItem("BACON"), getItem("BBQ_SAUCE"), getItem("BACON"), getItem("BACON"), new ItemStack(Material.COOKED_PORKCHOP), getItem("BACON"), getItem("CORNMEAL"), getItem("CHEESE"), getItem("CORNMEAL")},
+        20)
+        .register(plugin);
+        
+        new CustomFood(drinks, new SlimefunItemStack("SODA", "27c9c71f3328ec3a7daaf122aaadd65d69c2afc7738bc09519397b39c71da9a3", "&aSoda", "", "&7&oRestores &b&o" + "2.0" + " &7&oHunger"),
+        new ItemStack[] {getItem("ORANGE"), getItem("LIME"), getItem("LEMON"), new ItemStack(Material.SUGAR), new ItemStack(Material.WATER_BUCKET), getItem("ICE_CUBE"), null, null, null},
+        4)
+        .register(plugin);

```

Comment:
Again, random new item, no explanation, no outlining, no docs, not a single notice. And out of scope.